lib/openstudio/aws/aws.rb

```diff
@@ -25,9 +29,15 @@ def initialize(options = {})
         defaults = {
           ami_lookup_version: 1,
           region: 'us-east-1',
-          ssl_verify_peer: false
+          ssl_verify_peer: false,
+          save_directory: '.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.